src/plots/cartesian/graph_interact.js

```diff
@@ -60,7 +60,7 @@ fx.supplyLayoutDefaults = function(layoutIn, layoutOut, fullData) {
     coerce('dragmode');
 
     var hovermodeDflt;
-    if(layoutOut._hasCartesian) {
+    if(layoutOut._hasCartesian && !layoutOut._hasPie) {
```

Comment:
> Can we rip that out into its own thing

Good call.



> That's probably a bigger project

It should be pretty easy I think. I'll give it a shot in this PR.